.github/workflows/tests.yml

```
@@ -222,6 +222,7 @@ jobs:
       - name: Run brew test-bot ${{ needs.setup_tests.outputs.test-bot-formulae-args }}
         id: brew-test-bot-formulae
         run: |
+          [[ "$(uname -s)" = "Linux" ]] && rm -rf bottles
```

```suggestion
          rm -rf bottles
```

Let's just remove unconditionally. Even if not needed on macOS: better safe than sorry and it's not slow.

---

Ah, I did this just on Linux because I thought we want `mkdir bottles` to error out (at least on macOS) because we don't want to have stray bottles there for when @BrewTestBot tries to create bottle commits.